File: src/main/java/com/refinedmods/refinedstorage/render/RenderSettings.java
package com.refinedmods.refinedstorage.render;

import net.minecraft.ChatFormatting;

public class RenderSettings {
    public static final RenderSettings INSTANCE = new RenderSettings();

    private int primaryColor;
    private int secondaryColor;

    public RenderSettings() {
        setColors(-1, -1);
    }

    public void setColors(int primaryColor, int secondaryColor) {
        if (primaryColor == -1) {
            this.primaryColor = 4210752;
        } else {
            this.primaryColor = primaryColor;
        }

        if (secondaryColor == -1) {
            this.secondaryColor = ChatFormatting.WHITE.getColor();
        } else {
            this.secondaryColor = secondaryColor;
        }
    }

    public int getPrimaryColor() {
        return primaryColor;
    }

    public int getSecondaryColor() {
        return secondaryColor;
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/render/RenderSettings.kt
package com.refinedmods.refinedstorage.render

import net.minecraft.util.text.TextFormatting

class RenderSettings {
    var primaryColor = 0
        private set
    var secondaryColor = 0
        private set

    fun setColors(primaryColor: Int, secondaryColor: Int) {
        if (primaryColor == -1) {
            this.primaryColor = 4210752
        } else {
            this.primaryColor = primaryColor
        }
        if (secondaryColor == -1) {
            this.secondaryColor = TextFormatting.WHITE.getColor()
        } else {
            this.secondaryColor = secondaryColor
        }
    }

    companion object {
        @JvmField
        val INSTANCE = RenderSettings()
    }

    init {
        setColors(-1, -1)
    }
}